script/build-cli-docs

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
This script is a bit heinous. Would love to make it all happen more glamorously in a single Go script, but I'm not a good enough gopher to make that happen.

Halp!?

Reply:
Agreed that a separate `generate-docs` command, in a similar sense to `autocompletion` scripts would work well here. There's inspiration from the cobra docs themselves. a la https://www.emmanuelgautier.com/blog/snippets/go-cobra-docgen